### PR TITLE
BASE_IMAGE values are now picked up from robottelo.properties file

### PR DESCRIPTION
**Pull request by kbidarkar**
I tried running a test-case but it failed, may be I am not using the right combination of FF and selenium.
But as seen below, the VM can now be provisioned fine and it works for even Sat6.2.z branch.

(r62) [kbidarka@localhost robottelo]$ nosetests -m test_positive_install_package tests.foreman.ui.test_contenthost
EE
```
======================================================================
ERROR: Install a package to a host remotely
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/kbidarka/git_world/robottelo/tests/foreman/ui/test_contenthost.py", line 152, in test_positive_install_package
    with Session(self.browser):
  File "/home/kbidarka/git_world/robottelo/robottelo/ui/session.py", line 31, in __enter__
    self.login()
  File "/home/kbidarka/git_world/robottelo/robottelo/ui/session.py", line 40, in login
    self._login.login(self.user, self.password)
  File "/home/kbidarka/git_world/robottelo/robottelo/ui/login.py", line 21, in login
    self.click(common_locators['submit'])
  File "/home/kbidarka/git_world/robottelo/robottelo/ui/base.py", line 682, in click
    self.scroll_into_view(element)
  File "/home/kbidarka/git_world/robottelo/robottelo/ui/base.py", line 470, in scroll_into_view
    element,
  File "/home/kbidarka/.virtualenvs/r62/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 465, in execute_script
    'args': converted_args})['value']
  File "/home/kbidarka/.virtualenvs/r62/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/home/kbidarka/.virtualenvs/r62/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
WebDriverException: Message: arguments[0].scrollIntoView is not a function
Stacktrace:
    at anonymous (https://upgrade.redhat.com/users/login line 69 > Function:1)
    at handleEvaluateEvent (https://upgrade.redhat.com/users/login:69)
-------------------- >> begin captured logging << --------------------
robottelo: DEBUG: Started Test: ContentHostTestCase/test_positive_install_package
robottelo.ssh: DEBUG: Instantiated Paramiko client 0x7fe84d292450
robottelo.ssh: INFO: Connected to [provisoning_host.redhat.com]
robottelo.ssh: INFO: >>> snap-guest -b rhel72-base -t client.redhat.com -m 512 -c 1 -n bridge=br0 -f -p /opt/robottelo/images
robottelo.ssh: INFO: <<< stdout
Creating snapshot guest client.redhat.com
Creating snapshot image
Formatting '/opt/robottelo/images/client.redhat.com.img', fmt=qcow2 size=214748364800 backing_file='/var/lib/libvirt/images/rhel72-base.img' encryption=off cluster_size=65536 lazy_refcounts=off 
OS: linux rhel 7 2
Disable fsck startup check for all volumes
Configuring message of the day
Configuring /etc/hosts file
Setting up for Fedora / RHEL
Setting MAC address
Setting hostname
Provisioning guest client.redhat.com
Hostname: client.redhat.com
CPUs:     1
Memory:   512 MB
Swap:     512 MB
MAC:      52:54:00:53:2a:b2

Starting install...
Creating domain...                                          |    0 B  00:00     
Domain creation completed. You can restart your domain by running:
  virsh --connect qemu:///system start client.redhat.com

robottelo.ssh: DEBUG: Destroyed Paramiko client 0x7fe84d292450
robottelo.ssh: DEBUG: Instantiated Paramiko client 0x7fe84d297210
robottelo.ssh: INFO: Connected to [provisioning_host.redhat.com]
robottelo.ssh: INFO: >>> for i in {1..60}; do ping -c1 140635703684752.local && exit 0; sleep 1; done; exit 1
robottelo.ssh: INFO: <<< stdout
PING 140635703684752.local (10.257.257.257) 56(84) bytes of data.
64 bytes from dhcp-client.redhat.com (10.257.257.257): icmp_seq=1 ttl=64 time=0.310 ms

--- 140635703684752.local ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.310/0.310/0.310/0.000 ms

robottelo.ssh: INFO: <<< stderr
ping: unknown host 140635703684752.local

robottelo.ssh: DEBUG: Destroyed Paramiko client 0x7fe84d297210
robottelo.ssh: DEBUG: Instantiated Paramiko client 0x7fe84d292810
robottelo.ssh: INFO: Connected to [provisioning_host.redhat.com]
robottelo.ssh: INFO: >>> for i in {1..60}; do nc -vn 10.257.257.257 22 <<< "" && exit 0; sleep 1; done; exit 1
robottelo.ssh: INFO: <<< stdout
SSH-2.0-OpenSSH_6.6.1
Protocol mismatch.

robottelo.ssh: INFO: <<< stderr
Ncat: Version 6.45 ( http://nmap.org/ncat )
Ncat: Connected to 10.257.257.257:22.
Ncat: 1 bytes sent, 42 bytes received in 0.02 seconds.

robottelo.ssh: DEBUG: Destroyed Paramiko client 0x7fe84d292810
robottelo.ssh: DEBUG: Instantiated Paramiko client 0x7fe84d21ed10
robottelo.ssh: INFO: Connected to [10.257.257.257]
robottelo.ssh: INFO: >>> rpm -Uvh http://upgrade.redhat.com/pub/katello-ca-consumer-latest.noarch.rpm
robottelo.ssh: INFO: <<< stdout
Retrieving http://upgrade.redhat.com/pub/katello-ca-consumer-latest.noarch.rpm
Preparing...                          ########################################
Updating / installing...
katello-ca-consumer-upgrade-rh########################################
```

### CODE REVIEW

**Comment by kbidarkar** [comment]
More detailed, logs at http://pastebin.test.redhat.com/485940

**Comment by ldjebran** [comment]
Hi @kbidarkar, can you also add the commit from this PR also https://github.com/SatelliteQE/robottelo/pull/4529 
Thanks

File: robottelo/vm.py [hunk 1]
```diff
@@ -55,13 +50,21 @@ def __init__(
             self, cpu=1, ram=512, distro=None, provisioning_server=None,
             image_dir=None, tag=None, hostname=None, domain=None,
             target_image=None):
+        distro_el6 = settings.distro.image_el6
+        distro_el7 = settings.distro.image_el7
         self.cpu = cpu
         self.ram = ram
-        self.distro = BASE_IMAGES[-1] if distro is None else distro
-        if self.distro not in BASE_IMAGES:
+        if distro == 'rhel6':
```
Comment (kbidarkar) [inline robottelo/vm.py]: will fix this to use constants. I missed this.

Comment (kbidarkar) [inline robottelo/vm.py]: And all the below ones too.

Comment (ldjebran) [inline robottelo/vm.py]: Thanks

**Pull request merged**